crates/gen-host-js/src/lib.rs

```diff
@@ -1265,6 +1280,12 @@ impl Instantiator<'_> {
     }
 }
 
+#[derive(Copy, Clone)]
+enum Mode {
+    Internal,
+    External,
```

Comment:
Could this be renamed as "Lift" and "Lower" perhaps? I'm not sure what internal/external mean in this context myself.